wallet/subnet/primary/common/options.go

```diff
@@ -126,6 +130,10 @@ func (o *Options) PollFrequency() time.Duration {
 	return defaultPollFrequency
 }
 
+func (o *Options) LogTxIDFunc() LogTxIDFunc {
```

Comment:
Having this as on option feels kind of weird to me. Maybe we could have a more general `OnIssued` callback or something?